students/kangil/westagram/users/views.py

```diff
@@ -1,11 +1,13 @@
 import json
 import re
 import bcrypt
+import jwt
 
 from django.http import JsonResponse
 
 from django.views import View
 from users.models import User
+from my_settings import SECRET_KEY, ALGORITHM
```

Comment:
의존성을 줄이고자 `my_settings`에서 환경변수를 바로 불러오는 것이 아니라 `settings`로 부터 import 할 수 있도록 수정해보세요~
```suggestion
from django.views import View
from users.models import User
from my_settings  import SECRET_KEY, ALGORITHM
```